nextflow.config

```diff
@@ -280,11 +280,11 @@ manifest {
     contributors    = [
         // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
         [
-            name: 'kuebra.narci@dkfz.de',
+            name: 'kuebra.narci@dkfz-heidelberg.de',
             affiliation: '',
             email: '',
             github: '',
-            contribution: [], // List of contribution types ('author', 'maintainer' or 'contributor')
+            contribution: ['author', 'maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
             orcid: ''
```

Comment:
orcid is always nice to have.
You have no contributors on this pipeline?

Reply:
Is it okay to put this zenado now?
I only have reviewers :/

Reply:
the orcid is for you, (for example this is me: https://orcid.org/0000-0003-2827-9261)

Reply:
I have done this in another pipeline:
https://github.com/nf-core/references/blob/dev/nextflow.config#L228-L233